### DISCUSSION TITLE
draw dashed line?

### DISCUSSION THREAD

**Discussion by nfines** [QuestPDF/QuestPDF#592] [Q&A] [answered]
Looking at the chapter about line, I didn’t see that you can set the style of the line. Is there a way to draw a dashed line?

**Comment by girlpunk** [comment, marked as answer]
These can be drawn directly using SkiaSharp, as example of such is here

https://github.com/QuestPDF/QuestPDF/issues/577#issuecomment-1580325532

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
